commit 89be0b89ae60c63856fd26d82a104781540e2312
Author: Etsuro Fujita <efujita@postgresql.org>
Date:   Tue Aug 15 17:45:00 2023 +0900

    Fix code indentation vioaltion introduced in commit 9e9931d2b.
    
    Per buildfarm member koel

diff --git a/contrib/postgres_fdw/postgres_fdw.c b/contrib/postgres_fdw/postgres_fdw.c
index ef58b8b0ea2..1393716587e 100644
--- a/contrib/postgres_fdw/postgres_fdw.c
+++ b/contrib/postgres_fdw/postgres_fdw.c
@@ -6085,11 +6085,12 @@ add_paths_with_pathkeys_for_rel(PlannerInfo *root, RelOptInfo *rel,
 											 startup_cost,
 											 total_cost,
 											 useful_pathkeys,
 											 rel->lateral_relids,
 											 sorted_epq_path,
-											 NIL,	/* no fdw_restrictinfo list */
+											 NIL,	/* no fdw_restrictinfo
+													 * list */
 											 NIL));
 		else
 			add_path(rel, (Path *)
 					 create_foreign_join_path(root, rel,
 											  NULL,
@@ -6873,11 +6874,12 @@ add_foreign_ordered_paths(PlannerInfo *root, RelOptInfo *input_rel,
 											 rows,
 											 startup_cost,
 											 total_cost,
 											 root->sort_pathkeys,
 											 NULL,	/* no extra plan */
-											 NIL,	/* no fdw_restrictinfo list */
+											 NIL,	/* no fdw_restrictinfo
+													 * list */
 											 fdw_private);
 
 	/* and add it to the ordered_rel */
 	add_path(ordered_rel, (Path *) ordered_path);
 }
@@ -6989,11 +6991,12 @@ add_foreign_final_paths(PlannerInfo *root, RelOptInfo *input_rel,
 													   path->rows,
 													   path->startup_cost,
 													   path->total_cost,
 													   path->pathkeys,
 													   NULL,	/* no extra plan */
-													   NIL,	/* no fdw_restrictinfo list */
+													   NIL, /* no fdw_restrictinfo
+															 * list */
 													   NIL);	/* no fdw_private */
 
 				/* and add it to the final_rel */
 				add_path(final_rel, (Path *) final_path);
 
@@ -7110,11 +7113,11 @@ add_foreign_final_paths(PlannerInfo *root, RelOptInfo *input_rel,
 										   rows,
 										   startup_cost,
 										   total_cost,
 										   pathkeys,
 										   NULL,	/* no extra plan */
-										   NIL,	/* no fdw_restrictinfo list */
+										   NIL, /* no fdw_restrictinfo list */
 										   fdw_private);
 
 	/* and add it to the final_rel */
 	add_path(final_rel, (Path *) final_path);
 }
